GitHub: Improved infrastructure of win32metadata

0x5bfa opened the pull request:
## Summary

- Updated README.md to have a header and simplify links
- Added issue templates for bug report and question
- Added links to go CsWin32 and windows-rs before submitting to a wrong place
- Disabled blank issue
- Added pull request template (the same this this)

## Checklist

- [ ] Closes #xxx
- [ ] Built to validate changes

getrou commented on .github/PULL_REQUEST_TEMPLATE.md:
```diff
@@ -0,0 +1,8 @@
+## Checklist
+
+- [ ] Closes #xxx
+- [ ] Built to validate chnages
```
typo - changes

0x5bfa replied:
Thanks, fixed.

riverar commented:
Not sure about these changes, especially those that create more overhead in our issues/PR workflows (e.g., like requiring we check some box indicating we've tested locally).

riverar reviewed (changes requested):
Not sure about these changes, especially those that create more overhead in our issues/PR workflows (e.g., like requiring we check some box indicating we've tested locally).

I think we can limit the changes to doc tweaks and `.github` consolidation, but we'll need to double check the backend doesn't have any dependencies there.

0x5bfa commented:
I deleted PR template. Yet honestly I'd find issue templates very useful, what do you think? If not needed, I'd delete them.

I added the clear header in the readme and tweaked some.
